Helper/Data.php

```diff
@@ -263,20 +263,34 @@ public function cronSubscribeAll($list)
      */
     public function autoResponderAPiEmail($_data, $emailProduct)
     {
+        // send data to autoresponder limit 10 products
         $autoRespId = $this->getGeneralConfig('ac_ar_id');
-        $prod = @$emailProduct[0];
-
-        $address = [
-            'email' => $_data['email'],
-            'name' => $_data['customer_name'],
-            'abandoned_cart_url' => $this->getGeneralConfig('carturl'),
-        ];
         $response = false;
-        if (!empty($prod)) {
-            foreach ($prod as $field => $val) {
-                $address['product_' . $field] = $val;
+        if (!empty($emailProduct) && !empty($_data)) {
+            $address = [
+                'email' => $_data['email'],
+                'name' => $_data['customer_name'],
+                'abandoned_cart_url' => $this->getGeneralConfig('carturl'),
+            ];
+            //If more than one product in abandoned cart iterate to products array
+            if (count($emailProduct)>10) {
+                $address['over_10_products'] = true;
```

Comment:
Is there any special logic built around `over_10_products` variable in the Smaily template?

Reply:
It's supose to be an indicator that there were over 10 products in abandoned cart. You can't determine otherwise if user had 10 items in cart or more because current logic always limits to 10. With this indicator you could build logic around saying that you can find whole list of your products cart in cart_url if more than 10 is true or add some fancy faded out table etc.

Reply:
Changed value from boolean to text
https://github.com/sendsmaily/smaily-magento-extension/pull/12/commits/0cfb9706c277c4d2887914376049a2106ad5dbe3